### DISCUSSION TITLE
Eth0 as hotspot

### DISCUSSION THREAD

**Discussion by singersbalm** [RaspAP/raspap-webgui#1403] [Q&A]
I want to receive the connection over WiFi and output it over the built in lan. The hotspot feature does not work. How do I bridge the internet connection. The equivalent in iptables would be `iptables -t nat -A POSTROUTING -o eth0 -j MASQUERADE`. How do I set it up with raspap?

**Comment by billz** [comment]
Bridging wlan in client mode isn't allowed by the 802.11 wifi standard. 
The alternative is routing with NAT and IPv4 forwarding from wlan0 -> eth0. This isn't supported by RaspAP and there are no plans to implement it.

You might find some ideas [here](https://github.com/arpitjindal97/raspbian-recipes/blob/master/wifi-to-eth-route.sh). I wouldn't recommend executing this, however, as it deletes the contents of /etc/dnsmasq.d/*

**Reply by singersbalm** [reply]
@billz than you should filter it out in the ui, since it isn’t supported just hide it, so no one gets the wrong idea that it’s possible

**Reply by billz** [reply]
#1313 was created for this purpose

**Reply by billz** [reply]
@singersbalm we don't filter any interfaces, including loopback, but I can see how this might cause confusion for users. thanks for raising this

**Reply by billz** [reply]
Good cause for a new poll: https://twitter.com/rasp_ap/status/1708742043177029793
